lib/my_obfuscate/config_scaffold_generator.rb

```diff
@@ -0,0 +1,82 @@
+class MyObfuscate
+  module ConfigScaffoldGenerator
+
+    def generate_config(obfuscator, config, input_io, output_io)
+      input_io.each do |line|
+        if obfuscator.database_type == :postgres
```

Comment:
It'd be nice if we had a shared method to call in the `Mysql`, `Postgres`, etc. inclusion target that returns this so that we don't have to condition here.  I don't know if that would be any more / less clean, though.